src/fetchAPI.js

```diff
@@ -1,4 +1,4 @@
-const key = '2ee2ed8e-4950-4027-9dfb-a2c0e35f3218'
+const key = '643484ca-9138-4647-b058-430c60bdb96f'
```

Comment:
I don't know that this would have anything to do with the limit issues we've encountered, but a note I've heard and sounds like we need to consider - it's encouraged to set the API key to a variable as we've done, but move it to a file which gets added to the .gitignore so as to not be publically available.

Reply:
Yeah i like the idea of having a hidden folder with the key. Good call Allison!